lms/templates/edxnotes/note-item.underscore

```diff
@@ -17,7 +17,12 @@
     <ol class="note-comments" role="region" aria-label="<%- gettext('Note') %>">
         <li class="note-comment">
             <p class="note-comment-title"><%- gettext("You commented...") %></p>
-            <p class="note-comment-p"><%= text %></p>
+            <p class="note-comment-p">
+                <%= interpolate_text(_.escape(text), {
+                    highlighted_notes_start: '<span class="note-highlight">',
```

Comment:
@robrap  I would go for the first approach with renaming tags because it does match our rule to escape things at output to the last point(in templates). Also, this keeps the html `<span>` in templates instead of python (i don't like using/injecting html in python).